localization/pose_initializer/src/pose_initializer_core.cpp

```diff
@@ -50,29 +52,21 @@ PoseInitializer::PoseInitializer()
 {
   enable_gnss_callback_ = this->declare_parameter("enable_gnss_callback", true);
 
-  std::vector<double> initialpose_particle_covariance =
+  const std::vector<double> initialpose_particle_covariance =
     this->declare_parameter<std::vector<double>>("initialpose_particle_covariance");
-  for (std::size_t i = 0; i < initialpose_particle_covariance.size(); ++i) {
-    initialpose_particle_covariance_[i] = initialpose_particle_covariance[i];
-  }
+  CopyVectorToArray(initialpose_particle_covariance, initialpose_particle_covariance_);
```

Comment:
is it better lower camel case?

Reply:
Actually, `CamelCase` is allowed.
But considering no other package uses the naming rule, I feel `snake_case` or `camelCase` is better.

https://docs.ros.org/en/foxy/Contributing/Code-Style-Language-Versions.html#function-and-method-naming

Reply:
`for existing projects, prefer the existing style`

Do we have a plan to change the current naming rule?

Reply:
When we'll add packages to https://github.com/autowarefoundation/autoware.core, I believe we'll change the naming convention. :+1:

Reply:
OK. Then I keep this name